src/runtime/components/InputMenu.vue

```diff
@@ -86,24 +86,26 @@ export interface InputMenuProps<T> extends Pick<ComboboxRootProps<T>, 'modelValu
    * When `items` is an array of objects, select the field to use as the value instead of the object itself.
    * @defaultValue undefined
    */
-  valueKey?: keyof T
+  valueKey?: V
   /**
    * When `items` is an array of objects, select the field to use as the label.
    * @defaultValue 'label'
    */
   labelKey?: keyof T
-  items?: T[] | T[][]
+  items?: I
   /** Highlight the ring color like a focus state. */
   highlight?: boolean
   class?: any
   ui?: PartialString<typeof inputMenu.slots>
+  modelValue?: SelectModelValue<T, V, M>
```

Comment:
Maybe we can put back the TSDoc for `modelValue` and `multiple` here?

Reply:
Can you use the same TSDoc as the `ComboboxRoot` component for consistency?
![CleanShot 2024-10-28 at 10 50 33@2x](https://github.com/user-attachments/assets/b832f53c-1c71-4a76-bcf7-cf86c5ac4acc)